inc/composer/class-command.php

```diff
@@ -211,6 +220,8 @@ protected function get_env() : array {
 	protected function start( InputInterface $input, OutputInterface $output ) {
 		$output->writeln( '<info>Starting...</>' );
 
+		$this->check_ssl_certificate( $input, $output );
```

Comment:
I don't think we want to do this if the module config has `secure` set to false.

Reply:
Agreed 👍

Reply:
Actually, Traefik will still expect the SSL certificate to exist, so it's safer to just generate the file either way.

Reply:
Ah right. Hmm. That's based on the current static traefik.toml file right. I was more conscious of this code running on codespaces, not sure if it'll work or not there.